tough/Cargo.toml

```
@@ -9,21 +9,28 @@ keywords = ["tuf", "update", "repository"]
 edition = "2018"
 
 [dependencies]
+async-recursion = "1.0.5"
+async-trait = "0.1.73"
+bytes = "1.5.0"
 chrono = { version = "0.4", default-features = false, features = ["std", "alloc", "serde", "clock"] }
 dyn-clone = "1"
+futures = "0.3.28"
+futures-core = "0.3.28"
```

Can you reduce the specificity of the dependency versions? We have seen conflicts with other downstream dependencies at times when they require a "no-higher-than X" version.

```suggestion
async-recursion = "1"
async-trait = "0.1"
bytes = "1"
chrono = { version = "0.4", default-features = false, features = ["std", "alloc", "serde", "clock"] }
dyn-clone = "1"
futures = "0.3"
futures-core = "0.3"
```

---

Disregard, I will do this in a fast-follow as I work on publishing this! https://github.com/awslabs/tough/issues/699